pyscript.core/tests/integration/test_02_display.py

```diff
@@ -68,6 +72,7 @@ def test_target_parameter(self):
         mydiv = self.page.locator("#mydiv")
         assert mydiv.inner_text() == "hello world"
 
+    @skip_worker("NEXT: display(target=...) does not work")
```

Comment:
it does and it's tested ... maybe it's what we try to grab as element that doesn't work?